Resources/Prototypes/_Goobstation/Entities/Markers/Spawners/Random/posters.yml

```
@@ -0,0 +1,171 @@
+- type: entity
+  parent: MarkerBase
+  id: RandomPosterAny
+  name: random poster spawner
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)